### PR TITLE
update language of end of unknown element

### PR DESCRIPTION
**Pull request by dericed**
+ define descendant elements

### CODE REVIEW

**Comment by dericed** [comment]
intended to fix https://github.com/Matroska-Org/ebml-specification/issues/94

File: specification.markdown [hunk 1]
```diff
@@ -165,7 +167,7 @@ The `Element Data Size` expresses the length in octets of `Element Data`. The `E
...
...o one is reserved as an indicator that the size of the `EBML Element` is unknown. The only reserved value for the `VINT_DATA` of `Element Data Size` is all bits set to one. An `EBML Element` with an unknown `Element Data Size` is referred to as an `Unknown-Sized Element`. Only `Master Elements` SHALL be `Unknown-Sized Elements`. `Master Elements` MUST NOT use an unknown size unless the `unknownsizeallowed` attribute of their `EBML Schema` is set to true (see [the section on the `unknownsizeallowed` attribute](#unknownsizeallowed)). The use of `Unknown-Sized Elements` allows for an `EBML Element` to be written and read before the size of the `EBML Element` is known. `Unknown-Sized Element` MUST NOT be used or defined unnecessarily; however if the `Element Data Size` is not known before the `Element Data` is written, such as in some cases of data streaming, then `Unknown-Sized Elements` MAY be used. The end of an `Unknown-Sized Element` is determined by whichever comes first: the end of the file or the beginning of the next `EBML Element`, defined by this document or the corresponding `EBML Schema`, that is not independently valid as Descendent Element of the `Unknown-Sized Element`.
```
Comment (robUx4) [inline specification.markdown]: Should be back ticked here.

**Comment by robUx4** [comment]
Any reason you picked `Descendent` instead of `Child` ? As elements within a `Master` they might be called `Slave` but I think the master/slave metaphor is not considered insensitive.

Also my spell checker says it should be `Descendant`.

**Comment by dericed** [comment]
Descendant and Child are common in XML to reference relative element relationships.

For instance in:
```xml
<a>
  <b>
    <c/>
  </b>
</a>
```

<b> is a Child Element of <a>, but <c> is not. <b> and <c> are both Descendant Elements of <a>.

Since we had the phrase:
> The end of an `Unknown-Sized Element` is determined by the beginning of the next element, defined by this document or the corresponding `EBML Schema`, that is not a valid sub-element of the `Unknown-Sized Element`.

I wanted to make it clear that if <a> is unknown sized, that the start of <c> does not imply that it is the end of <a>, even though <c> may not be a valid Child of <a>.

**Comment by dericed** [comment]
changed `Descendent` to `Descendant`, and backticked `Descendant Element`

**Pull request merged**